tests/validation/cattlevalidationtest/core/test_ldap.py

```diff
@@ -0,0 +1,181 @@
+from common_fixtures import *  # NOQA
+from requests.auth import AuthBase
+
+
+if_ldap = pytest.mark.skipif(not os.environ.get('API_AUTH_LDAP_SERVER'),
+                             reason='API_AUTH_LDAP_SERVER is not set')
+
+
+BASE_URL = cattle_url()
```

Comment:
I see you use this BASE_URL when posting for tokens. Are tokens not a normal part of the API that you could use the client to create?